tecan_od_analyzer/tecan_od_analyzer.py

```diff
@@ -276,7 +274,7 @@ def compensation_lm(cor_df, df_gr) : #done
 	linear = lambda x, a, b: a * x + b
 	sns.set(style="white", palette="muted", color_codes=True)
 	fig = plt.figure()
-	fig.suptitle('Linear models of volume loss correlation to time for different plates', fontweight="bold")
+	fig.suptitle('Volume loss correlation over time', fontweight="bold")
 	#sns.despine(left=True)
```

Comment:
If this line is not needed, I would recommend removing it.  If it might be needed later at some point, I would recommend adding that into the comment.